src/openms/include/OpenMS/ANALYSIS/OPENSWATH/DIAHelper.h

```diff
@@ -48,8 +49,16 @@ namespace OpenMS
 
     /**
       @brief Integrate intensities in a spectrum in range @p im_range (if defined) for multiple windows.
-    */
+      @p windows_center is a vector of the center location of the windows.
+      @p width is the width of the windows across mz
+      @p RangeMobility is the range of the IM dimension (if defined)
+
+      Returns:
+      integrated_windows_intensity is a vector of the integrated intensity for each window
+      integrated_windows_mz is a vector of the integrated intensity-weighted m/z for each window
+      integrated_windows_im is a vector of the integrated intensity-weighted im for each window
```

Comment:
use `@param` or (even better) `@param[out] ...`  (note: no space between `@param` and `[out]`

Reply:
also document `remove_zero`